cspell-wordlist.txt

```diff
@@ -0,0 +1,31 @@
+!PostCss
```

Comment:
This is normally stylized as `PostCSS` but we had `PostCss` several places in the codebase, so I'm disallowing that here with the `!`

See <https://cspell.org/docs/dictionaries-custom/#words-list-syntax>

Reply:
(we could do this with other things too in the future, like making sure we don't have `Nodejs` instead of `NodeJS` or something, just wanted to have one example of it in the first PR here)